var/spack/repos/builtin/packages/genie/package.py

```diff
@@ -0,0 +1,195 @@
+    version("2.12.6", sha256="3b450c609875459798ec98e12cf671cc971cbb13345af6d75bd6278d422f3309")
+    version("2.12.4", sha256="19a4a1633b0847a9f16a44e0c74b9c224ca3bb93975aecf108603c22e807517b")
+    version("2.12.2", sha256="cbdc45a739878940dadcaaed575b5cad6b5e7035f29605045b1ca557e6faa6d1")
+    version("2.12.0", sha256="d2b01c80f38d269cb0296b3f2932798ef3f1d51bd130e81274fbfeeb381fac6b")
+    version("2.11.2", sha256="0f4c25d8ceb7513553671643c9cdac5aa98c40fc8594a5ecb25c077c6b36166e")
+    version("2.11.0", sha256="1ebe0eb65d797595413632f1cec1cb2621cb8e8d0384a2843799724a79b1d80c")
+    version("2.10.10", sha256="1dfaadcf1bbaf6e164b612f410c4399301e63497ad6a4891706b1787ac11a7a1")
+    version("2.10.8", sha256="4f6f5af2062e7c505b76e70547ac2ae304a9790c3e9b9592818d8aebeebc8398")
+    version("2.10.6", sha256="d00b4288c886f81459fb2967e539f30315d4385f82d1d3f4330298d313f9a176")
+    version("2.10.4", sha256="df909bf7e1a789ca01794995687da2af803769f0823273a4a3a31678d6d5b0f1")
+    version("2.10.2", sha256="6abe4e0cdb5e8f5beddf0ccdbebc94c175a9f72592b1cbbffe01b88ee3972bf9")
+    version("2.10.0", sha256="17bda900c996b6f4f10a7f6a3be94e56c3b8dcdeb2ef8865ca7f20c5fe725291")
+    version("2.9.0", sha256="8229beb73f65f5af86a77bf141acfbe4a8b68cba9d797aae083a929906f6f2a2")
+    version("2.8.6", sha256="310dc8e0d17a65e6b9773e398250703a3a6f94ceafe94f599ae0f7b3fecf7e6c")
+
+    depends_on("root+pythia6")
+    depends_on("pythia6")
+    depends_on("lhapdf", when="@3:")
+    depends_on("lhapdf5", when="@:2")
+    depends_on("log4cpp")
+    depends_on("libxml2")
+    depends_on("gsl")
+
+    # GENIE does not actually require cmake, but root does.
+    # Spack's concretizer fails with "unsatisfiable constraint" if we don't add this.
+    depends_on("cmake@3:")
+
+    # GENIE Makefile's think that the spack compiler is invalid.
+    # Disables this check.
+    patch("genie_disable_gopt_with_compiler_check.patch", level=0, when="@2.11:")
+
+    # Flags for GENIE"s optional but disabled by default features
+    variant(
+        "vleextension",
+        default=False,
+        description="Enable GENIE very low energy (1 MeV - 100 MeV) extension",
+    )
+    variant(
+        "validationtools",
+        default=False,
+        description="GENIE physics model validation tools",
+    )
+    variant("test", default=False, description="Enable test programs")
+    variant("t2k", default=False, description="Enable T2K-specific generation app")
+    variant("fnal", default=False, description="Enable T2K-specific generation app")
+    variant(
+        "atmo",
+        default=False,
+        description="Enable GENIE Atmospheric neutrino event generation app",
+    )
+    variant(
+        "nucleondecay",
+        default=False,
+        description="Enable GENIE Nucleon decay event generation app",
+    )
+    variant(
+        "masterclass",
+        default=False,
+        description="Enable GENIE neutrino masterclass app",
+    )
```

Comment:
Is there a particular reasoning behind the order for these variants? For improved readability I recommend sorting them. (It's just for readability for developers: variants are always sorted when displayed through spack's interface.)

Reply:
No reason for the ordering. I have sorted them in 5eaee4d.